demos/gesture_recognition_demo/cpp_gapi/src/exponential_averager.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "exponential_averager.hpp"
+
+ExponentialAverager::ExponentialAverager(double smoothingFactor, double initValue):
+                     smoothingFactor(smoothingFactor), value(initValue) {
+}
+
+double ExponentialAverager::getAveragedValue() const {
+    return value;
+}
+
+void ExponentialAverager::updateValue(double newValue) {
+    value = smoothingFactor * newValue + (1. - smoothingFactor) * value;
+}
```

Comment:
This thing looks quite tiny, it is used everywhere or just in a single place? If the latter, I'd drop a separate file here and put it right to the place where it is used.

Reply:
Will be removed.